libs/jsonforms-components/src/lib/Controls/AddressLookup/AddressLookUpControl.tsx

```diff
@@ -112,40 +119,120 @@ export const AddressLookUpControl = (props: AddressLookUpProps): JSX.Element =>
     setSuggestions([]);
     setErrors({});
   };
+
```

Comment:
@Sankeerth-Mulkalla-goa 
Can you add this line
`  /* istanbul ignore next */ `

on top of the function handleRequiredFieldBlur to ignore this block of code for the code coverage